### DISCUSSION TITLE
Exclude Object type for IED

### DISCUSSION THREAD

**Discussion by Vill63** [Vdauphin/HeartsAndMinds#1347] [Question and answer]
Hello,
I'd like to port H&M to Unsung but I can see that some IEDs are created with pickup/cars wrecks which does not fit with the era. How I can I exclude these 2 types of objects ?
Best regards,

Vill

**Comment by Vdauphin** [comment]
Hello,

You can add them to the list to remove here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L357
If I remember, it is case sensitive

Cheers
